neo/rawio/blackrockrawio.py

```diff
@@ -438,8 +438,7 @@ def _parse_header(self):
             # Find maximal and minimal time for each nev segment
             for k, (data, ev_ids) in self.nev_data.items():
                 for i in np.unique(ev_ids):
-                    mask = [ev_ids == i]
-                    curr_data = data[mask]
+                    curr_data = data[ev_ids == i]
```

Comment:
This looks good to me.  (There is no other usage of 'mask' in scope so this simplification seems right.)